bin/libfreenect2/depends/INSTALL-windows.txt

```diff
@@ -0,0 +1,49 @@
+Source code: https://github.com/OpenKinect/libfreenect2
+API reference: http://openkinect.github.io/libfreenect2
+
```

Comment:
- [ ] I don't believe this file is needed.

Reply:
The following files/directories were accidentally added by Eclipse users. Then someone noticed that and added them to .gitignore but they didn't clean up before. I just cleaned up the following files/folders and it should solve this issue:

bin
.classpath
.settings
.project